ext/scheduler/airflow/__lib.py

```diff
@@ -400,7 +400,7 @@ def get_run_type(context):
 def job_success_event(context):
     try:
         meta = {
-            "event_type": "TYPE_JOB_SUCCESS",
```

Comment:
is this change expected?

Reply:
yes this is needed

Reply:
No, actually you are right, this should not be done